apps/developer-hub/src/components/search-button.tsx

```diff
@@ -0,0 +1,32 @@
+"use client";
+
+import DefaultSearchDialog from "fumadocs-ui/components/dialog/search-default";
+import { useState } from "react";
+
+export const SearchButton = () => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <>
+      <button
+        onClick={() => {
+          setOpen(true);
+        }}
+        style={{
+          background: "transparent",
+          border: "1px solid #ccc",
+          padding: "8px 16px",
+          borderRadius: "6px",
+          cursor: "pointer",
+        }}
+      >
+        Search
+      </button>
```

Comment:
you should use the `<Button>` component from the component library, e.g. try to match something like this from IH: https://github.com/pyth-network/pyth-crosschain/blob/main/apps/insights/src/components/Root/search-button.tsx#L73-L99